CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Yarn now accepts sponsorships! Please give a look at our [OpenCollective](https:
 
 ### **Major Changes**
 
+- With Node.js 16's End of Life [approaching fast](https://nodejs.org/en/blog/announcements/nodejs16-eol), we dropped support for both Node.js 14 and 16, with Node.js 18 being the recommended LTS version.
```

Comment:
```suggestion
- With Node.js 16's End of Life [approaching fast](https://nodejs.org/en/blog/announcements/nodejs16-eol), we dropped support for Node.js 12, 13, 14, 15, 16, and 17, with Node.js 18 being the recommended LTS version.
```
There isn't a changelog entry for it but we also dropped support for <14.15 in https://github.com/yarnpkg/berry/pull/4196.